defaults.json

```diff
@@ -296,13 +619,88 @@
         "Weasel",
         "Weese",
         "Wenda",
+        "Werlag",
         "Wilbert",
-        "Will Humble",
         "Will",
+        "Will Humble",
+        "Willam",
         "William",
+        "Willit",
+        "Willum",
+        "Wolmer",
         "Woth",
+        "Wulfe",
+        "Wylla",
         "Yandry",
         "Yna",
-        "Zei"
-    ]
+        "Yorkel",
+        "Ysilla",
+        "Zei",
+        "Zekko",
+        "Zollo"
+    ],
+
+    "sentimentlist": {
+        "honorable": 2,
+        "nothing": -1,
+        "fck": -4,
+        "fck u": -4,
+        "coolest": 2,
+        "lord": 3,
+        "princess": 3,
+        "boss": 3,
+        "humble": 2,
+        "smug": -2,
+        "feast": 2,
+        "woo hoo": 3,
```

Comment:
we use a tokenizer which splits at every whitespace. So this world could never be matched.